src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Draw {
+    public void setParameters(int sideA,int sideB,int height,int firstLeg,
+                           int secondLeg,int radius, String color) {
+        /*int sideA;
+        int sideB;
+        int height;
+        int firstLeg;
+        int secondLeg;
+        int radius;*/
+    }
+
+    public String behaviour() {
+        return null;
+    }
```

Comment:
lets remove this methods, we dont need them. Instead lets create here `color` field with setters